code/modules/awaymissions/away_props.dm

```diff
@@ -51,4 +58,72 @@
 			if(blocked_types[thing.type])
 				return reverse
 	return !reverse
-	
+
+// Away Mission Rework Items
+
+/obj/item/awaymaploader
+	name = "debug away mission loader - report this!"
+	desc = "A disk containing a set of data codes needed to lock onto an away mission. Insert it into the station gateway to lock onto the mission."
+	icon = 'icons/obj/module.dmi'
+	icon_state = "datadisk0"
+	w_class = WEIGHT_CLASS_TINY
+	var/unstable = FALSE // Does the map have a time limit?
```

Comment:
Can't you just check if var/time is not zero instead of using additional variable?